StPetersburg/RNBspbdaBIII11.xml

```diff
@@ -0,0 +1,805 @@
+                                    <title type="complete">Abraham</title>
+                                </msItem>
+
+                                <msItem xml:id="ms_i2.119">
+                                    <title type="complete">John the Baptist</title>
+                                </msItem>
+
+                            </msItem>
+                        </msItem>
+
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">66</measure>
+                                    <measure unit="leaf" type="blank">3</measure>
+                                    <locus target="#1r #1v #2r #2v #66r #66v"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>132</height>
+                                        <width>92</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="11"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                             <seg type="script" cert="medium">18th-century script.</seg>
+                            </handNote>
+
+                            <handNote script="Cyrillic" xml:id="h2" corresp="#e1"/>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            <decoNote type="ornamentation" xml:id="d1">
+                                <locus target="#3r"></locus>
+                                <desc>Ornamental band on the incipit page (lineal ornament)</desc>
+                            </decoNote>
+                            
+                            <decoNote type="ornamentation" xml:id="d2">
+                               <desc>Ornamental bands drawn with black and red inks, at the end of the parts of the text</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="Comment">A comment concerning the content (<ref target="ms_i2">"Zǝmmāre chants</ref>)</desc>
+                                    <locus target="#1r"/>
+                                    <q xml:lang="ru">Cerkovnyja pěsnopěnija dlya svjaščennikov" i diakonov" otdělno</q>
```

Comment:
cyrillic, or type="normalized"

Reply:
Do you mean like this: `<q xml:lang="ru" type="normalized">`?

Reply:
yes, if the transcription is provided for non-Latin-script languages I would expect it to be specified. otherwise the ru range is for cyrillic Церковныя пѣснопѣния для священниковъ и диаконовъ отдѣлно

Reply:
I did the first option.